nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1836,39 +1741,14 @@ define([
             }
         }
 
-        // FSM state change events
-
-        function doStartRunning() {
+        function updateJobState() {
```

Comment:
generic function to get the current job status and generate a job status line using the `Jobs` module.